### ISSUE TITLE
Amazon blocked/ ban account.

### ISSUE THREAD

**Issue by devNostyle** [Hari-Nagarajan/fairgame#556]
Bot run well,  but today wanted some capatcha. I think ok - np.
The problem come bigger - on my amazon account **only** in chrome browser i  cannot go to to my account settings and cannot display ""see all buing options" in tabs.

i got this msg;

"We’re sorry. The Web address you entered is not a functioning page on our site."

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chorme 
 - Version [89]

on firefox runs normally, on moblie to i  connact to support on  amazon - they dk how fix ...

I reinstall chrome, clean registry, try on other computer ....
But when i log on 2nd amazon account it's work well.

**Comment by DakkJaniels** [comment]
We don't troubleshoot problems with your Amazon account.

**Comment by H4TERAID** [comment]
Same here, can't see any buying options from other sellers, I think we are banned from using this function?

**Comment by Ampzx** [comment]
Same here. Even get dogs when clicking on account. Still able to order stuff though. Just not able to pull up sellers on sidebar or go to account

**Comment by BigCTBug** [comment]
yep also here, i start to use it yesterday run it through the night and after 4 hour i got "Item is currently unavailable. moving on..." and cant load the side pop up

**Comment by devNostyle** [comment]
Now u must contact with amazon support - only they can resolve problem.
It's little ban.

**Comment by Ampzx** [comment]
> Now u must contact with amazon support - only they can resolve problem.
> It's little ban.

Have you done this already? Now it works? Can you still use bot without ban?

**Comment by devNostyle** [comment]
> 
> 
> > Now u must contact with amazon support - only they can resolve problem.
> > It's little ban.
> 
> Have you done this already? Now it works? Can you still use bot without ban?

Ofc not - u get ban again.  Futhrermore using this probably u  never get any item.
